CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ rumqttc v0.19.0
 - Fix examples to stop printing error in loop (#540)
 - MQTTv5!: Remove `Connect` from `ConnectionError::StateError` (#541)
 - MQTTv5: Send last_will and login info with connect (#478)
+- MQTTv5: Read the Unsubscribe package in match arms (#625)
```

Comment:
this changelog is for old release, instead of here can you add it in unreleased section [here](https://github.com/bytebeamio/rumqtt/blob/main/rumqttc/CHANGELOG.md)

Reply:
@henil is it fine now ?